spec/tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+
+KEY_1_FPR=A845A5BD622556E89D7763B5EB06D1696BEC4C90
+
+test_package_signature 0.9.1 "${KEY_1_FPR}"
```

Comment:
What about using `git ls-remote --tags https://github.com/rnpgp/rnp` or something like this here, so future tags will be picked as well?